packages/Webkul/Sales/src/Database/Factories/InvoiceFactory.php

```diff
@@ -60,8 +51,7 @@ public function definition(): array
             'base_tax_amount'       => $taxAmount,
             'discount_amount'       => 0,
             'base_discount_amount'  => 0,
-            'order_id'              => $attributes['order_id'],
-            'order_address_id'      => $attributes['order_address_id'],
+            'order_id'              => Order::factory(),
```

Comment:
Remove this factory dependency. No factory will depend on each other.

Reply:
done